cluster/src/main/java/io/servicefabric/cluster/gossip/GossipRequest.java

```diff
@@ -7,7 +7,7 @@
 /**
  * Gossip request which be transmitted through the network, contains list of gossips
  */
-final class GossipRequest {
+public final class GossipRequest {
```

Comment:
Maybe better to move benchmark tests to the corresponding packages since I am considering Gossip and GossipRequest part of internal implementation, wdyt?